products/azul-zulu.md

```diff
@@ -0,0 +1,211 @@
+    latestJdkVersion: "16.0.2+7"
+    latestReleaseDate: 2021-07-24
+    link: https://docs.azul.com/core/zulu-openjdk/release-notes/july-2021
+
+-   releaseCycle: "15"
+    releaseLabel: "15 (<abbr title='Medium Term Support'>MTS</abbr>)"
+    lts: false
+    releaseDate: 2020-09-15
+    eol: 2023-03-31
+    extendedSupport: 2024-03-31
+    latest: "15.46.17"
+    latestJdkVersion: "15.0.10+5"
+    latestReleaseDate: 2023-01-17
+    link: https://docs.azul.com/core/zulu-openjdk/release-notes/january-2023
+
+-   releaseCycle: "14"
+    releaseLabel: "14 (<abbr title='Short Term Support'>STS</abbr>)"
+    lts: false
+    releaseDate: 2020-03-17
+    eol: 2020-09-30
+    extendedSupport: 2021-03-31
+    latest: "14.29.23"
+    latestJdkVersion: "14.0.2+12"
+    latestReleaseDate: 2020-07-17
+
+-   releaseCycle: "13"
+    releaseLabel: "15 (<abbr title='Medium Term Support'>MTS</abbr>)"
+    lts: false
+    releaseDate: 2019-09-17
+    eol: 2023-03-31
+    extendedSupport: 2024-03-31
+    latest: "13.54.17"
+    latestJdkVersion: "13.0.14+5"
+    latestReleaseDate: 2023-01-17
+    link: https://docs.azul.com/core/zulu-openjdk/release-notes/january-2023
+
+-   releaseCycle: "12"
+    releaseLabel: "12 (<abbr title='Short Term Support'>STS</abbr>)"
+    lts: false
+    releaseDate: 2019-03-19
+    eol: 2019-09-30
+    extendedSupport: 2020-03-31
+    latest: "12.3.11"
+    latestJdkVersion: "12.0.2+3"
+    latestReleaseDate: 2019-07-16
+
+-   releaseCycle: "11"
+    lts: true
+    releaseDate: 2018-09-25
+    eol: 2026-09-30
+    extendedSupport: 2028-09-30
+    latest: "11.62.17"
+    latestJdkVersion: "11.0.18+10"
+    latestReleaseDate: 2019-07-16
+    link: https://docs.azul.com/core/zulu-openjdk/release-notes/january-2023
+
+-   releaseCycle: "10"
+    releaseLabel: "10 (<abbr title='Short Term Support'>STS</abbr>)"
+    lts: false
+    releaseDate: 2018-03-20
+    eol: 2018-09-30
+    extendedSupport: 2019-03-31
+    latest: "10.3.5"
+    latestJdkVersion: "10.0.2+13"
+    latestReleaseDate: 2018-07-26
+
+-   releaseCycle: "9"
+    releaseLabel: "9 (<abbr title='Short Term Support'>STS</abbr>)"
+    lts: false
+    releaseDate: 2017-09-21
+    eol: 2018-03-31
+    extendedSupport: 2018-09-30
+    latest: "9.0.7.1"
+    latestJdkVersion: "9u7"
+    latestReleaseDate: 2018-04-27
+
+-   releaseCycle: "8"
+    lts: true
+    releaseDate: 2014-03-18
+    eol: 2030-12-31
+    extendedSupport: 2032-12-31
+    latest: "8.68.0.21"
+    latestJdkVersion: "8u362-b09"
+    latestReleaseDate: 2023-01-24
+    link: https://docs.azul.com/core/zulu-openjdk/release-notes/january-2023-8.68
+
+-   releaseCycle: "7"
+    lts: true
+    releaseDate: 2011-07-11
+    eol: 2022-07-31
+    extendedSupport: 2027-12-31
+    latest: "7.56.0.11"
+    latestJdkVersion: "7u352-b01"
+    latestReleaseDate: 2022-07-19
+    link: https://docs.azul.com/core/zulu-openjdk/release-notes/july-2022
+
+-   releaseCycle: "6"
+    lts: true
+    releaseDate: 2006-12-12
```

Comment:
As per https://www.azul.com/newsroom/azul-systems-extends-zulu-to-support-java-6-and-major-linux-distributions/, Java6 support came late.

(Wikipedia had this incorrect, fixed it as well)

```suggestion
    releaseDate: 2014-01-21
```

Reply:
Ok, updated. Should have read all your comments before searching ;).